icgcget/clients/gdc/gdc_client.py

```diff
@@ -17,49 +17,55 @@
 #
 
 import re
-
+import tempfile
 from icgcget.clients.errors import ApiError
 from icgcget.clients.download_client import DownloadClient
 from icgcget.clients.portal_client import call_api
 
 
 class GdcDownloadClient(DownloadClient):
 
-    def __init__(self, pickle_path=None):
+    def __init__(self, pickle_path=None, verify=True):
         super(GdcDownloadClient, self) .__init__(pickle_path)
         self.repo = 'gdc'
+        self.verify = verify
 
-    def download(self, uuids, access, tool_path, output, processes, udt=None, file_from=None, repo=None, region=None):
+    def download(self, uuids, access, tool_path, output, processes, udt=None, file_from=None, repo=None, password=None):
         call_args = [tool_path, 'download']
         call_args.extend(uuids)
         call_args.extend(['--dir', output, '-n', processes])
+
         if access is not None:  # Enables download of unsecured gdc data
-            call_args.extend(['-t', access])
+            access_file = tempfile.NamedTemporaryFile()
+            access_file.file.write(access)
+            access_file.file.seek(0)
+            call_args.extend(['--token', access_file.name])
```

Comment:
No, partially because the references to that variable got removed in the most recent update to gdc's github page.  

https://github.com/NCI-GDC/gdc-client/commit/0211df8409b3bbd1adcd91ed30d7fa223365a6dc